bindings_node/src/signatures.rs

```diff
@@ -112,6 +112,33 @@ impl Client {
     Ok(signature_text)
   }
 
+  #[napi]
+  pub async fn revoke_installations_signature_text(
```

Comment:
I can't remember why we needed a separate function for the signature text instead of using the `SignatureRequest` returned from `revoke_installations`. I'm sure there was a good reason.